modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -16,8 +16,45 @@
 	position: fixed;
 }
 
-.dropdown-menu.date-picker-dropdown-menu.show {
-	position: fixed;
+@media screen and (max-height: 850px) {
+	.dropdown-menu.date-picker-dropdown-menu.show {
+		height: 21rem;
+		position: fixed;
+	}
+}
+
+@media screen and (max-height: 750px) {
+	.dropdown-menu.date-picker-dropdown-menu.show {
+		height: 15rem;
+		position: fixed;
+	}
+}
+
+@media screen and (max-height: 550px) {
+	.dropdown-menu.date-picker-dropdown-menu.show {
+		height: 10rem;
+		position: fixed;
+	}
+}
+
+@media screen and (max-height: 450px) {
+	.dropdown-menu.date-picker-dropdown-menu.show {
+		height: 8rem;
+		position: fixed;
+	}
+}
+
+@media screen and (max-height: 370px) {
+	.dropdown-menu.date-picker-dropdown-menu.show {
+		height: 6rem;
+		position: fixed;
+	}
+}
+
+@media screen and (min-height: 851px) {
+	.dropdown-menu.date-picker-dropdown-menu.show {
+		position: fixed;
+	}
```

Comment:
Hey @icgsbr! I think this one should be the first media query, since it is related to the higher resolution possible you know? But since this is urgente, I won't ask you to fix this now, I can do it myself later. Thanks!